content/applications/inventory_and_mrp/purchase/manage_deals/control_bills.rst

```diff
@@ -1,60 +1,117 @@
-   :align: center
-   :alt: Vendor bills default control setting for new products in Odoo Purchase
+Example flow: Received quantities
+=================================
 
-By selecting *On ordered quantities*, you can create a vendor bill as soon as you confirm an order.
-The quantities mentioned in the purchase order are used to generate the draft bill.
+To complete an example workflow using the *Received quantities* bill control policy, first go to
+:menuselection:`Purchase --> Configuration --> Settings`, scroll down to the *Invoicing* section,
+and select :guilabel:`Received quantities`. Then, :guilabel:`Save` changes.
 
-If you choose *On received quantities* instead, you can only create one once you have at least
-received some of the products you have ordered. The quantities you have received are used to
-generate the draft bill. If you try to create one without having received any product, you get an
-error message.
+In the Purchase app, create a new Request for Quotation. Fill out the information on the quotation,
+add products to the invoice lines, and click :guilabel:`Confirm Order`. Then, click on the
+:guilabel:`Receipt smart button`. Set the quantities in the *Done* column to match the quantities
+in the *Demand* column, and :guilabel:`Validate`. Then, in the purchase order, click
+:guilabel:`Create Bill` and :guilabel:`Confirm`. Since the policy is set to *Received quantities*,
+the *Draft Bill* can be confirmed *only* when at least some of the quantities are received.
 
-.. image:: control_bills/control-bills-no-invoiceable-line.png
-   :align: center
-   :alt: No invoiceable line error message in Odoo Purchase
+3-way matching
+==============
 
-Change a specific product's control policy
-------------------------------------------
+Activating 3-way matching ensures that vendor bills are only paid once some or all of the products
+included in the purchase order have actually been received. To activate it, go to
+:menuselection:`Purchase --> Configuration --> Settings`, and scroll down to the **Invoicing**
+section. Then, click :guilabel:`3-way matching: purchases, receptions, and bills`.
 
-If you want to modify a specific product's control policy, go to :menuselection:`Products -->
-Products`, open it, click on *Edit* and go to the *Purchase tab*. There you can change a product's
-default bill control policy.
+.. note::
+   3-way matching is intended to work with the bill control policy set to *Received quantities*.
+
+Know when to pay vendor bills with 3-way matching
+-------------------------------------------------
 
-.. image:: control_bills/control-bills-quantities-on-product.png
+When 3-way matching is activated, vendor bills will display the :guilabel:`Should Be Paid` field
+under the :guilabel:`Other Info` tab. When a new vendor bill is created, the field will be set to
+*Yes*, since a bill can't be created until at least some of the products included in a purchase
+order have been received.
```

Comment:
```suggestion
When :guilabel:`3-way matching` is activated, vendor bills will display the :guilabel:`Should Be
Paid` field under the :guilabel:`Other Info` tab. When a new vendor bill is created, the field will
be set to :guilabel:`Yes`, since a bill can't be created until at least some of the products
included in a purchase order have been received.
```